src/co_aberration.jl

```diff
@@ -0,0 +1,99 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _co_aberration{T<:AbstractFloat}(jd::T, ra::T, dec::T, eps::T)
+    t = (jd -2451545)/36525
+    if isnan(eps)
+        eps0 = ten(23,26,21.448)*3600 - 46.815*t - 0.00058*(t^2) +
+               0.001813*(t^3)
+        eps = sec2rad(eps0 + nutate(jd)[2])
+    end
+    sunlong = T(sunpos(jd, radians=true)[3])
```

Comment:
It shouldn't be necessary to convert the result to `T`, hopefully `sunpos` already returns a `T` element, doesn't it?